docs/reference/tags/variant_pricing/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Variant Pricing
+parent: Tags
+grand_parent: Reference
+---
+
+# Variant Pricing
+
+The `variant_pricing` tag renders a self-updating `<span>` which can
+interactively fetch and display up-to-date pricing information for a given
+variant, provided an adult count/start date/end date combination.
+
+You can use this tag to display up-to-date prices of each variant you're
+rendering on the page, responding to user interactions with the calendar and
+adult count for each displayed variant.
+
+The tag is package-aware; if you use it in a package step template, it will
+display how much upgrading to that variant will cost, or `Included` if it is
+part of the package price.
+
+##### input
+{% raw %}
+
+    {% variant_pricing variant: variant, class: "my-nice-class", adult_count: 2, start_on: "2023-01-01", end_on: "2023-01-04" %}
+
+{% endraw %}
+
+##### output
+
+The `<span>` tag will initially be rendered empty:
```

Comment:
I'm not sure this is correct? On inital page load, this should be returning a result based on the values initially passed in.

Reply:
Yup I mistook the use of `DOMContentLoaded` to bootstrap the content for the tag's actual behavior. I see that we've already talked about tweaking the tag so it actually does this -- let me check what comes out of standup and I'll feed back on this.

Reply:
Okay, I'll update this to reflect the current behavior then revisit the docs once we fix the tag.

Reply:
Addressed 6861ff5